src/components/tasks/crud/createTask.js

```diff
@@ -57,60 +57,70 @@ const basicSwal = {
 
 const MySwal = withReactContent(Swal)
 
-export const createTask = (tasks, reload, setReload) => {
+export const createTask = (addTask, reload, setReload) => {
```

Comment:
addTask raczej nie jest dobrą nazwą zmiennej na to. Co najwyżej taskToAdd

Reply:
To nie jest task do dodania, tylko funkcja typu `Task => Promise` która ma się wykonać po wciśnięciu guzika stworzenia taska